docs/src/main/sphinx/connector/opensearch.md

```diff
@@ -311,60 +371,21 @@ It is not allowed to use `asRawJson` and `isArray` flags simultaneously for the
 
 The following hidden columns are available:
 
-| Column   | Description                                         |
-|----------|-----------------------------------------------------|
-| \_id     | The OpenSearch document ID                          |
-| \_score  | The document score returned by the OpenSearch query |
-| \_source | The source of the original document                 |
-
-## Predicate push down
-
-The connector supports [predicate push down](predicate-pushdown) for the following data types:
-
-| OpenSearch   | Trino         | Supported     |
-|--------------|---------------|---------------|
-| `binary`     | `VARBINARY`   | `NO`          |
```

Comment:
I dropped the ones with NO from the table and added a sentence after the table that nothing else is supported.